tests/template/test_template_schema.json

```diff
@@ -18,6 +18,13 @@
         "TES"
       ]
     },
+    "versions": {
+      "type": "array",
+      "description": "The list of TES versions against which the test file can be run",
```

Comment:
Let's not introduce any new bindings to TES. So better replace with "TES" with "API".

Reply:
Corrected this.